cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,148 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'lara_c',
+    password: 'Password!1'
+  };
 
+  const book = {
+    title: 'Speaking JavaScript',
+    author: 'Axel Rauschmayer',
+    publisher: 'O\'Reilly Media',
+    description: 'Like it or not, JavaScript is everywhere these day'
+  };
+
+  const allertMessage = {
+    added: 'Book added to your collection.',
+    deleted: 'Book deleted.'
+  };
+
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it.only('should allow to login', () => {
+    cy.findByPlaceholder('UserName')
+      .type(user.username);
+
+    cy.findByPlaceholder('Password')
+      .type(user.password);
+
+    cy.get('#login')
+      .click();
+
+    cy.url()
+      .should('include', '/profile');
+
+    cy.get('#userName-value')
+      .should('contain', user.username);
+
+    cy.contains('button', 'Log out')
+      .should('be.visible');
+  });
+
+  it.only('should allow to add a book to user collection', () => {
+    cy.login(user.username, user.password);
```

Comment:
Calling a function `cy.login(user.username, user.password)`, but I cannot find where this function is defined. If it's a custom command, it should be defined in the commands file.